lib/webview-helpers.js

```diff
@@ -481,11 +487,14 @@ helpers.createChromedriverCaps = function createChromedriverCaps (opts, deviceId
   if (opts.pageLoadStrategy) {
     caps.pageLoadStrategy = opts.pageLoadStrategy;
   }
-  if (_.toLower(caps.chromeOptions.androidPackage) === 'chrome') {
+  if (_.includes(KNOWN_CHROME_PACKAGE_NAMES, caps.chromeOptions.androidPackage)
+    || _.toLower(caps.chromeOptions.androidPackage) === 'chrome') {
```

Comment:
maybe can define `const isChrome = _.toLower(caps.chromeOptions.androidPackage) === 'chrome'` outside `if` clause and use it in here and the below?

Reply:
Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/a6ec697256e163bd5faa2e04dba730eb4f156c86